tests/bdd/test_volume_create.py

```diff
@@ -0,0 +1,195 @@
+"""Volume creation feature tests."""
+import os
+import subprocess
+import time
+
+from pytest_bdd import (
+    given,
+    scenario,
+    then,
+    when,
+)
+
+import pytest
+import docker
+import requests
+
+import common
+
+from openapi.openapi_client.model.create_pool_body import CreatePoolBody
+from openapi.openapi_client.model.create_volume_body import CreateVolumeBody
+from openapi.openapi_client.model.volume_spec import VolumeSpec
+from openapi.openapi_client.model.protocol import Protocol
+from openapi.openapi_client.model.spec_state import SpecState
+from openapi.openapi_client.model.volume_state import VolumeState
+from openapi.openapi_client.model.volume_status import VolumeStatus
+
+VOLUME_UUID = "5cd5378e-3f05-47f1-a830-a0f5873a1449"
+VOLUME_SIZE = 10485761
+NUM_VOLUME_REPLICAS = 1
+REST_SERVER = "http://localhost:8081/v0"
+CREATE_REQUEST_KEY = "create_request"
+POOL_UUID = "4cc6ee64-7232-497d-a26f-38284a444980"
+NODE_NAME = "mayastor"
+
+
+# This fixture will be automatically used by all tests.
+# It starts the deployer which launches all the necessary containers.
+# A pool is created for convenience such that it is available for use by the tests.
+@pytest.fixture(autouse=True)
+def init():
+    common.deployer_start(1)
+    common.get_pools_api().put_node_pool(
+        NODE_NAME, POOL_UUID, CreatePoolBody(["malloc:///disk?size_mb=500"])
```

Comment:
malloc uses hugepages, do we really need 500mb?

Reply:
Reduced all of these to 50mb.